### PR TITLE
Poster1186

### PR DESCRIPTION
**Pull request by andersonwinkler**

### CODE REVIEW

File: posters-overrides.json [hunk 1]
```diff
@@ -3339,7 +3339,13 @@
   },
   {
    "number": 1186
```
Comment (effigies) [inline posters-overrides.json]:
```suggestion
   "number": 1186,
```

File: posters-overrides.json [hunk 2]
```diff
@@ -3339,7 +3339,13 @@
...
-  },
+   "title": "Permutation Inference for Canonical Correlation Analysis (CCA)",
+   "institution": ["National Institutes of Health (NIH)", "University of Geneva", "University of Oxford"],
+   "presenter": "Anderson M. Winkler",
+   "categories": ["Statistics", "Modelling", "Analysis"],
+   "pdf": "https://s3.us-east-2.amazonaws.com/brainder/publications/2020/SOHBM2020-PermCCA.pdf",  
+   "authors": [ "Anderson M. Winkler", "Olivier Renaud", "Stephen M. Smith", "Thomas E. Nichols" ],
+   "keywords": [ "permutation tests", "canonical correlation analysis", "multivariate statistics"] }
```
Comment (effigies) [inline posters-overrides.json]:
```suggestion
   "keywords": [ "permutation tests", "canonical correlation analysis", "multivariate statistics"]
  },
```

**Review by effigies — COMMENTED** [review]
Some quick cleanups.

File: posters-overrides.json [hunk 3]
```diff
@@ -3338,7 +3338,14 @@
    "number": 1184
   },
   {
-   "number": 1186
+   "number": 1186,
+   "title": "Permutation Inference for Canonical Correlation Analysis (CCA)",
+   "institution": ["National Institutes of Health (NIH)", "University of Geneva", "University of Oxford"],
```
Comment (effigies) [inline posters-overrides.json:3343]: I'm not sure if the code is set up to handle string fields being overridden with lists, so I will leave further review to those more involved in the recent changes.

Comment (yarikoptic) [reply posters-overrides.json:3343]: yes, please limit only to PDF URL -- affiliation of the speaker already picked up, all the authors as well.

Comment (soichih) [reply posters-overrides.json:3343]: I just sent https://github.com/datalad-datasets/ohbm2020-posters/pull/129 that can handle institution field to be either string or array of strings.

**Comment by yarikoptic** [comment]
#131 needs to be addressed before the merge so we could test

**Comment by andersonwinkler** [comment]
Feel free to drop everything while keeping just the PDF. I think I copied from an example but maybe that wasn't right... Thanks, this is a beautiful work!

**Comment by yarikoptic** [comment]
I will add override locally for PDF only and push (along with my WiP script which shouldn't ruin anything)

**Comment by andersonwinkler** [comment]
Argh! Sadly the URL has a typo... the correct one, for poster 1186 is: https://s3.us-east-2.amazonaws.com/brainder/publications/2020/OHBM2020-PermCCA.pdf
Thanks!

**Pull request closed without merging**